package com.java;

public class ArithmaticException {

	public static void main(String[] args) {
		 int a = 7;
		 try {
			 int result = 7/0;
		} catch (Exception e) {
		//System.out.println(e.getMessage());
			e.printStackTrace();
		
	}
	}
}
